.pre-commit-config.yaml

```diff
@@ -71,6 +71,12 @@ repos:
         exclude: ^src/em/emoji.*\.json$
         exclude_types: [yaml]
 
+  - repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v1.14.1
+    hooks:
+      - id: mypy
+        files: '^src/em/'
```

Comment:
Let's include `tests/` too, perhaps by removing `files:` and excluding `scripts/`?